src/index.js

```diff
@@ -321,25 +321,26 @@ class ReactDadata extends React.Component {
       placeholder,
       showNote,
       styles,
+      customInput,
     } = this.props;
 
     const showSuggestionsList = inputFocused && showSuggestions;
 
+    const inputConfig = {
+      className: `react-dadata__input${allowClear ? ' react-dadata__input-clearable' : ''}`,
+      placeholder: placeholder,
+      value: query,
+      autoComplete: autocomplete || 'off',
+      ref: input => { this.textInput = input },
```

Comment:
unused code ref.

**PS:** if we need make normal refs for customInputs and use from locally (but this is bad idea),it  need to use forwardRef wrapping